operator/v1/types_ingress.go

```diff
@@ -308,6 +316,32 @@ type EndpointPublishingStrategy struct {
 	NodePort *NodePortStrategy `json:"nodePort,omitempty"`
 }
 
+// RouteAdmissionPolicy is an admission policy for allowing new route claims.
+type RouteAdmissionPolicy struct {
+	// namespaceOwnership describes how host name claims across namespaces should
+	// be handled. The default is Strict.
+	//
+	// Value must be one of:
+	//
+	// - Strict: do not allow routes to claim the same host name across namespaces.
+	// - InterNamespaceAllowed: allow routes to claim different paths of the same host name across namespaces.
+	//
+	// The default is Strict.
+	NamespaceOwnership NamespaceOwnershipCheck `json:"namespaceOwnership"`
```

Comment:
Remind me whether you're handling this by interpreting empty to mean "Strict" (which I think is safe for downgrade too) or if you're relying CRD defaulting to set a value of "Strict", which I think could have issues with update order if the ingresscontroller (client) updates first and expects the value to be present, but the CRD hasn't yet updated.

Reply:
I went with "interpreting empty to mean Strict"